omod/src/main/webapp/resources/scripts/fragments/patientdashboard/encountertemplate/defaultEncounterTemplate.js

```diff
@@ -19,23 +20,25 @@ $(function() {
         var patientId = $(event.target).attr("data-patient-id");
         var actionUrl = $(event.target).attr("data-edit-url") || $(event.target).attr("data-view-url");
         var dataMode = $(event.target).attr("data-mode");
+        var uiStyle = fixUiStyle($(event.target).attr("data-ui-style"));
```

Comment:
A nitpick, but probably better to call this "setUiStyle" or "formatUiStyle"... I get what you mean by "fix", but it's probably not if people don't go into it knowing what it's supposed to do.

Reply:
Thanks for the comment @mogoodrich . I made changed function name to getUiStyle and and brought all the resolving attribute part into the function itself. 

Let me know yours thoughts.